data/data/openstack/masters/private-network.tf

```diff
@@ -102,7 +102,7 @@ resource "openstack_networking_port_v2" "api_port" {
 
     content {
       subnet_id  = fixed_ip.value["subnet_id"]
-      ip_address = var.openstack_api_int_ip
+      ip_address = var.openstack_api_int_ips[0]
```

Comment:
why just the first?

Reply:
I should have read your comment first!

> Note that the creation of the Neutron Ports for the VIPs only uses the first VIP for now. The reason behind it is that it needs to be handled differently when using either BYON or basic installation, and it will be handled in separate PR.